go/vt/vtgate/vindexes/vschema.go

```diff
@@ -788,9 +880,37 @@ func FindVindexForSharding(tableName string, colVindexes []*ColumnVindex) (*Colu
 	return result, nil
 }
 
-// ToString prints the table name
-func (t *Table) ToString() string {
+func (t *Table) MarshalJSON() ([]byte, error) {
```

Comment:
Created this so `(*vindexes.Table).Source` can be a different type than `(proto.VSchema).Source`.